src/Microsoft.OData.Edm/Csdl/CsdlReader.cs

```diff
@@ -293,7 +307,7 @@ private static bool TryParseVersion(string input, out Version version)
             return true;
         }
 
-        private bool TryParse(IEnumerable<IEdmModel> referencedModels, out IEdmModel model, out IEnumerable<EdmError> parsingErrors)
+        private bool TryParse(IEnumerable<IEdmModel> referencedModels, out IEdmModel model, out IEnumerable<EdmError> parsingErrors, bool enableVocabularySupport = true)

```

Comment:
>, bool enableVocabularySupport = true [](start = 131, length = 37)

Rather than add an optional param (which breaks compiled apps trying to call this method), I would add a new overload with enableVocabularySupport in its proper place (before any output parameters -- i know it works to put an input parameter after an output parameter but, well, yuck.)

Reply:
Resolved. I forget to remove the default value.

Resolved.

---
In reply to: [252881416](https://github.com/OData/odata.net/pull/1367#discussion_r252881416) [](ancestors = 252881416)

Reply:
Just finished the clean up. Thanks.

---
In reply to: [252895152](https://github.com/OData/odata.net/pull/1367#discussion_r252895152) [](ancestors = 252895152,252889714,252881416)